docs/getting-started/zns-emulation.md

```diff
@@ -35,10 +35,7 @@ create a sparse file or by using the *dd* command to create a fully allocated fi
 
 #### Creating the Backstore
 
-This guide provides instructions for two methods of creating the backstore file:
-
-1. Using truncate
-2. Using dd
+This guide provides instructions for creating the backstore file using ``truncate``.
 
 ##### Using truncate to create an Emulated Zone Namespace Backstore
```

Comment:
I do not see the point of this sub-header now that the other one is removed. Let's drop it.